lib/ui/nav_pages/dm_page/widgets/dmmessage_read.dart

```diff
@@ -9,7 +12,9 @@ class DMMessageRead extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return InkWell(
-      onTap: () {},
+      onTap: () {
+        locator<NavigationService>().navigateTo(Routes.dmUserView);
```

Comment:
Move this logic away from the view, and put in the viewmodel, also follow the above request